src/handlers.rs

```diff
@@ -1405,7 +1405,7 @@ impl Handler {
         {
             platform = "ios".to_string();
         }
-        let mut feature = "".to_string();
+        let feature = "".to_string();
```

Comment:
The problem here is, i am overwriting this in the features, that isnt recognized by the linters.

Reply:
Good point. One can run the linter with `--all-features` or add an exception here.